src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -193,5 +221,18 @@ internal void Validate()
                 throw new InvalidOperationException($"{nameof(this.MaxConcurrentOrchestratorFunctions)} must be a non-negative integer value.");
             }
         }
+
+        // Returns true if WEBSITE_SLOT_NAME is Production or when WEBSITE_SLOT_NAME is not set.
```

Comment:
In general, I am of the belief that non-documentation comments should describe WHY the code is written this way, not WHAT the code this was doing. This comment reads more like a documentation comment, in that it says what the method does. For private methods, the name should ideally be descriptive enough to not require a documentation comment.

I would move the WHY parts of this comment (i.e. non Azure Functions environment likely won't have "WEBSITE_SLOT_NAME" as an environment variable) closer to the code itself, and not put it at the method level.

Reply:
I actually meant this as that documentation-type method summary comment. I can include a comment like you suggested in code instead, though, if you think that would be best.